### PR TITLE
Update devDeps

### PR DESCRIPTION
**Pull request by Vijar**
@redonkulus

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: package.json [hunk 1]
```diff
@@ -28,17 +28,17 @@
...
+    "chai": "^3.0.0",
     "coveralls": "^2.11.1",
     "express": "^4.12.3",
     "istanbul": "^0.3.2",
     "jshint": "^2.5.1",
     "mocha": "^2.0.1",
     "mockery": "^1.4.0",
-    "precommit-hook": "^1.0.2",
-    "qs": "^2.4.1",
+    "precommit-hook": "^2.0.0",
```
Comment (redonkulus) [inline package.json]: use pre-commit, the latest version is 1

**Comment by Vijar** [comment]
@redonkulus updated.

**Comment by redonkulus** [comment]
:+1:

**Pull request merged**